modules/ROOT/pages/apicat-create-descriptor-file-cli.adoc

```diff
@@ -67,6 +67,36 @@ The presence and content of the designated descriptor file affects the result of
 
 * Existing file with content: The command fails and shows an error message that says the descriptor file already exists.
 
+=== Add Images through a Descriptor File
```

Comment:
I would add the `and link documentation pages` if we want to document both use cases together. If not, we can probably have two different sections